### DISCUSSION TITLE
Info Stolen/Leaked

### DISCUSSION THREAD

**Discussion by doomdoge** [community/community#22689] [New to GitHub]
<p>I just got my info stolen and leaked from a Github website. Please help me. The owner of the website: <a href="https://github.com/Julie420" rel="nofollow noopener">https://github.com/Julie420</a></p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hello <a class="mention" href="/doomdoge">@doomdoge</a>,</p>
<p>Thank you for being here and for reaching out!</p>
<p>If you haven’t already, you’ll need to contact our <a href="https://support.github.com/contact/account">private support</a> team with the info for further assistance. You will also see more detail here on <a href="https://docs.github.com/en/github/building-a-strong-community/reporting-abuse-or-spam">reporting abuse</a>.</p>

**Comment by doomdoge** [comment]
<p>Thank you so much for the help!</p>

**Comment by doomdoge** [comment]
<p>The user has the ip logger linked on their profile page</p>
